text_utils/utils.py

```diff
@@ -1,17 +1,19 @@
 import logging
+import os
 import re
 
 
 def create_logger(logger_name: str, logger_path: str) -> logging.Logger:
     logger = logging.getLogger(logger_name)
     logger.setLevel(logging.DEBUG)
-    formatter = logging.Formatter('%(asctime)s %(message)s')
+    formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
     file_handler = logging.FileHandler(filename=logger_path)
     file_handler.setFormatter(formatter)
     stream_handler = logging.StreamHandler()
     stream_handler.setFormatter(formatter)
     logger.addHandler(stream_handler)
     logger.addHandler(file_handler)
+    os.makedirs('logs', exist_ok=True)
```

Comment:
How to be sure we in the right dir? It makes directory related to current importing script path

Reply:
To be fixed in separate patch